### DISCUSSION TITLE
Its easy to code with tailwind?

### DISCUSSION THREAD

**Discussion by LuisFernandoPBPereira** [luanayasmim/portfolio-aline#5] [Q&A] [answered]
Like, i think that's okay, but is it difficult?

**Comment by luanayasmim** [comment, marked as answer]
No, is very simple, just add names in the tag class and things just happen

**Comment by adarsh01122** [comment]
hy i m using tailwind css and having multiple issues , first tags are not being rendered , then colors or any css property is not rendered in webpage , i have tried every possible steps  , 
<img width="919" alt="Screenshot 2024-03-28 184728" src="https://github.com/luanayasmim/portfolio-aline/assets/146781753/eab45c07-418b-400d-83de-f87a1c726ab1">
